tests/ppx/main_deriving.ml

```diff
@@ -0,0 +1,16 @@
+module Type_conv = Ppx_type_conv.Type_conv
+
+let l = Ppx_deriving.derivers ()
+let _ = ListLabels.iter l ~f:(fun (deriver : Ppx_deriving.deriver) ->
+  Type_conv.add deriver.Ppx_deriving.name
```

Comment:
@diml: `type_conv` derivers can be used with `ppx_deriving` using `ppx_type_conv.deriving`.
What about `ppx_deriving` derivers used with `type_conv` ?